libs/features/personalization/personalization.js

```diff
@@ -131,20 +131,45 @@ const CREATE_CMDS = {
   appendtosection: 'beforeend',
 };
 
+const COMMANDS_KEYS = {
```

Comment:
I don't feel this array is necessary, since it's a 1 to 1 with the keys in COMMANDS.

Reply:
After discussing it, I see the value more.  No longer concerned.